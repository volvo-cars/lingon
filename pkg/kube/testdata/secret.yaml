apiVersion: v1
data:
  bla: Ymxh
kind: Secret
metadata:
  name: super-secret