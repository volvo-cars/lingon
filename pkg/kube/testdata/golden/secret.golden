&v1.Secret{
	Data:       map[string][]byte{"bla": []byte("<REDACTED>")},
	ObjectMeta: v11.ObjectMeta{Name: "super-secret"},
	TypeMeta: v11.TypeMeta{
		APIVersion: "v1",
		Kind:       "Secret",
	},
} // TODO: SECRETS SHOULD BE STORED ELSEWHERE THAN IN THE CODE!!!!